main.rb

```diff
@@ -0,0 +1,40 @@
+require './src/app'
+
+def present_options
+  puts "
+  Choose an option:
+
+  1. List Items
+  2. Add a new Book
+  3. Add a new Music album
+  4. Add a new Movie
+  5. Add a new Game
+  6. Exit"
```

Comment:
- [ ] I am still getting an error when exiting the app, check if you have made a typo on the name below 
![image](https://user-images.githubusercontent.com/62705947/202705870-be11fe4b-6d73-49f2-9b92-1ce2d11f8c4b.png)

Reply:
I have no errors like this in my local machine

Reply:
OH i see the linters missed my code

Reply:
- [ ] It is coming from this file, meaning that I don't have a folder it needs to create one but it is throwing an error. It can be that you already have a folder and that is why is not throwing an error to you 
![image](https://user-images.githubusercontent.com/62705947/202707098-952097c1-41b2-4c42-9b3b-ef31a616ebbd.png)

Reply:
accually I didn't right this piece at all FileUtils 
It's automatically added after I done the auto lint  rubocop -A
and I didn't test the app after 
but thank you the advice though

Reply:
No problem, keep up the great work :clap: